ios/FluentUI/People Picker/PeoplePickerTokenSet.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `PeoplePicker` control.
+public class PeoplePickerTokenSet: ControlTokenSet<PeoplePickerTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The background color of the PeoplePicker.
+        case backgroundColor
+    }
+
+    init() {
+        super.init { token, theme in
+            switch token {
```

Comment:
nit: Feels weird to have a switch with a single element to iterate over. Could this be rewritten differently?

Reply:
It seems like some other controls also do it this way (`Separator`, `PersonaButtonCarousel`). I'm assuming it also makes it easier to add new tokens but would you still prefer it to be rewritten?

Reply:
I guess then it's fine but maybe something that's worth bringing up for potential cleanup in the future.